src/main/controlMod.F90

```diff
@@ -259,6 +259,17 @@ subroutine control_init( )
     namelist /clm_inparm/ n_dom_pfts
     namelist /clm_inparm/ n_dom_landunits
 
+    ! Thresholds above which the model keeps the soil, crop, glacier, lake,
+    ! wetland, and urban landunits
+    namelist /clm_inparm/ toosmall_soil
```

Comment:
This is a nitpick, but you don't have to have each of these on their own line. They could be comma delimited on a line. For grepability it might be good to not NOT & to extend. IF you prefer this syntax, that's fine, I mostly want to check so you know it's not required.

Reply:
No strong preference, so I extended.